metal/datasource_metal_facility.go

```diff
@@ -59,7 +58,7 @@ func dataSourceMetalFacilityRead(d *schema.ResourceData, meta interface{}) error
 				"features": f.Features,
 				"metro": func(d *schema.ResourceData, k string) error {
 					if f.Metro != nil {
-						return d.Set(k, f.Metro.Code)
+						return d.Set(k, toLower(f.Metro.Code))
```

Comment:
same as https://github.com/equinix/terraform-provider-metal/pull/126/files#r650953584

I think `toLower` makes sense for the statefuncs, but it adds indirection in the `d.Set` arguments.

Reply:
OK, I changed toLower to strings.ToLower in relevant places in the code